### DISCUSSION TITLE
AttributeError: module 'torchio' has no attribute 'ScalarImage'

### DISCUSSION THREAD

**Discussion by r2chel** [TorchIO-project/torchio#554] [Q&A]

**Comment by fepegar** [comment]
Hi, @r2chel. Instead of editing your question to leave it empty, could you please share your problem and how you solved it?

**Comment by r2chel** [comment]
Sorry, I tried to delete it entirely because I think the issue was on my end (things not being updated etc). I keep getting an attribute error saying:

AttributeError: module 'torchio' has no attribute 'ScalarImage'

When I tried to run the code posted here: https://stackoverflow.com/questions/64674612/how-to-resize-a-nifti-nii-gz-medical-image-file

with my own image. I didn't want to post until I could put enough useful information.
***@***.***?v=73d79a89bded]<https://stackoverflow.com/questions/64674612/how-to-resize-a-nifti-nii-gz-medical-image-file>
How to resize a nifti (nii.gz medical image) file - Stack Overflow<https://stackoverflow.com/questions/64674612/how-to-resize-a-nifti-nii-gz-medical-image-file>
I have some medical images of nii.gz format which are of different shapes. I want to resize all to the same shape inorder to feed to a deep learnig model, I tried using resample_img() of nibabel, b...
stackoverflow.com



Rachel Chua | Undergraduate Research Assistant

Undergraduate Neuroscience Program

UAB | The University of Alabama at Birmingham

P: 256.658.8537 | ***@***.******@***.***>
***@***.***>


________________________________

**Reply by fepegar** [reply]
You might have an old version of `torchio`. Could you please check your installed version with `print(tio.__version__)`?

**Reply by fepegar** [reply]
Also, it would be helpful if you could provide a minimal working example to reproduce your error.

**Comment by r2chel** [comment]
[cid:7a0b50f5-9676-46ee-a970-2aa2adba9d27]

Found existing installation: torchio 0.18.34



Rachel Chua | Undergraduate Research Assistant

Undergraduate Neuroscience Program

UAB | The University of Alabama at Birmingham

P: 256.658.8537 | ***@***.******@***.***>
***@***.***>


________________________________

**Comment by r2chel** [comment]
I upgraded to 0.18.39 and am having the same issue.


Rachel Chua | Undergraduate Research Assistant

Undergraduate Neuroscience Program

UAB | The University of Alabama at Birmingham

P: 256.658.8537 | ***@***.******@***.***>
***@***.***>


________________________________

**Reply by fepegar** [reply]
Please open a new issue using the Bug report template and follow all the instructions, so we can help you.
